nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -49,6 +49,8 @@
 
 - [Conduwuit](https://conduwuit.puppyirl.gay/), a federated chat server implementing the Matrix protocol, forked from Conduit. Available as [services.conduwuit](#opt-services.conduwuit.enable).
 
+- [COSMIC DE](https://system76.com/cosmic/), the COSMIC desktop environment by Pop!_OS.
+
```

Comment:
Feel free to bump this to highlights. A new desktop environment is very notable.